### ISSUE TITLE
IINA's new icon blends into the Dock in Dark Mode.

### ISSUE THREAD

**Issue by pleasejustgo** [iina/iina#3216]
I liked the white border of the old icon. It's not a huge deal it's just that the new icon blends into the Dock in Dark mode.

**Comment by imaverage** [comment]
is this in Big Sur?

**Comment by pleasejustgo** [comment]
no, Mojave. But I can't see how it would be any different in Big Sur. The dock in Dark mode should have the same dark color in it for any version of macOS right? And the color of IINA's icon simply blends into the Dock dark mode color.

**Comment by imaverage** [comment]
Interesting. I have catalina in dark mode but the dock doesn't seem very dark. The icon looks fine, if not gorgeous.
